lib/src/resources/app_assets.dart

```diff
@@ -2,4 +2,5 @@ final class AppAssets {
   const AppAssets._();
 
   static const imagesLogo = 'assets/images/logo.png';
+  static const fakeAvatarImage = 'assets/images/fake_avatar.jpg';
```

Comment:
```suggestion
  static const imagesFakeAvatar = 'assets/images/fake_avatar.jpg';
```
ça doit respecter la convention qui est de reprendre le nom du fichier mais en camelCase (sauf assets et l'extension)

Reply:
J'ai retirer ce code je ne l'utilisait pas pour cette branche actuelle